### ISSUE TITLE
keras on tensorflow  how to define customers loss function

### ISSUE THREAD

**Issue by xianyongzhang** [keras-team/keras#8701]
I want to define a customers loss function ：
for example 

def myloss(self,y_true,y_pred):
      tshape = y_true.shape
      
      y_true_2d = tf.reshape(y_true, [ tshape[0]*tshape[1]  tshape[2]])     tell me  error!!!!!
      I want to reshape y_true to [ tshape[0]*tshape[1]  tshape[2]] , but how to implement it ?
      also I want     svd decomposition y_true_2d  but how to implement it ???
     is there any way to call function in tensorflow?

Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [ ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).
